code/web/src/modules/user/Profile.js

```diff
@@ -57,15 +57,17 @@ const Profile = ({ user, logout }) => {
       <Grid>
         <GridCell style={{ padding: "2em", textAlign: "center" }}>
           <img
-            src={details.image ? `${APP_URL_API}/images/uploads/${details.image}` : `${APP_URL}//images/stock/men/4.jpg`}
+            src={(details.image !== null) ?`${APP_URL_API}/images/uploads/${details.image}` : `${APP_URL}/images/stock/men/4.jpg` }
```

Comment:
I don't think this change was necessary, but it should still work the same either way